src/natcap/invest/seasonal_water_yield/seasonal_water_yield.py

```diff
@@ -163,31 +163,31 @@
                 "Required if args['monthly_alpha'] is false.  Is the "
                 "proportion of upslope annual available local recharge that "
                 "is available in month m."),
-            "name": "alpha_m Parameter"
+            "name": "alpha_m parameter"
```

Comment:
If we're using UTF-8, would it be worth using `𝛼ₘ` here, and is that sort of formatting something that we should care about?

Reply:
sure, I do like the unicode variable characters! Weirdly that subscript m won't display for me in my browser or text editor, or in the workbench. So these will have to keep using the underscore notation.

Reply:
Bummer, but I'm also kind of surprised that UTF-8 wouldn't render properly.  This makes me wonder if there's a deeper issue with how the characters would be rendering.

In any case, let's leave `alpha_m` as-is and I'll create a separate issue on the workbench repo to clarify this.